app/templates/gentelella/guest/event/_ticketing_box.html

```diff
@@ -270,9 +270,9 @@ <h3 class="panel-title">{{ _("Ticket Information") }}</h3>
         var discount_amount = -1;
         var promo_code_status = 0;
         var discount_code = "";
-        {% if code %}
+        {% if discount_code %}
```

Comment:
no I'm not, the issue was that both access code and discount code use the same query parameter `code` so when user uses access code link he cannot use discount code, The issue is to separate them and we can use access code in future commits.